.github/workflows/rust.yml

```
@@ -8,72 +8,55 @@ on:
     paths-ignore:
       - '**.md'
 
-env:
-  CARGO_TERM_COLOR: always
-
 jobs:
   build:
     strategy:
       matrix:
         include:
         - name: windows-latest
-          runs-on: [self-hosted, Windows]
+          runs-on: [self-hosted, 25.05]
```

```suggestion
          runs-on: [self-hosted, Windows, 25.05]
```
When the linux self-hosted runner is added we'll need the OS tag also.